frontend/src/pages/places.vue

```diff
@@ -272,6 +272,14 @@ export default {
       }).catch(() => this.loading = false);
     },
     renderMap() {
+      // In case of facing RTL language on the map, this plugin will be fetched (only by lazy!) and will set RTL properly support.
+      // See mapbox documentation here https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-rtl-text/
+      mapboxgl.setRTLTextPlugin(
+        'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.3/mapbox-gl-rtl-text.js',
```

Comment:
I wonder if that file could be integrated in PhotoPrism? It would remove the dependency on an external URL that can fail.

Reply:
The map content is fetched on run-time from an external API in any case, and integrating it into PhotoPrism will increase the frontend bundle.

Reply:
Fair enough. Thanks.

Reply:
It's somewhat of a privacy and stability issue indeed, like when their server is down or blocked by a firewall. We could store a copy of that file in our static assets folder and load it from there?

https://github.com/photoprism/photoprism/tree/develop/assets/static

Reply:
@lastzero you're right, see the #1398 PR with this improvement.